pkgs/by-name/vz/vzic/package.nix

```
@@ -0,0 +1,51 @@
+{
+  stdenv,
+  lib,
+  fetchFromGitHub,
+  pkg-config,
+  glib,
+  tzdata,
+  libical,
+}:
+stdenv.mkDerivation rec {
+  pname = "vzic";
+  version = "2024a";
+
+  src = fetchFromGitHub {
+    owner = "libical";
+    repo = "vzic";
+    rev = "4a87e2bed237585cbcf209507d0b82c023113633";
+    hash = "sha256-zRy62SJa8XUBPnoW/fgANKZaatNWqmXka4LgRUlbLzw=";
+  };
+
+  nativeBuildInputs = [
+    pkg-config
+    tzdata
+    glib
+    libical
+  ];
```

These dependencies are not split correctly. (`strictDeps = true` could enforce proper separation here)

- pkg-config is invoked at built time, so it needs to go to `nativeBuildInputs`
- glib gets compiled & linked against, so it needs to go to `buildInputs`

tzdata & libical don't seem to make any difference on the build result. I see that libical at least is present in upstream's hardcoded FHS include flags, so maybe it's included in case it's ever needed? Not sure about tzdata, please decide on your own if it's really needed & which input it needs to go into - I'll leave it as-is for the suggestion.

The Makefile also uses a hardcoded `pkg-config`, which we don't always have.
```suggestion
  postPatch = ''
    substituteInPlace Makefile \
      --replace-fail 'pkg-config' "$PKG_CONFIG"
  '';

  nativeBuildInputs = [
    pkg-config
    tzdata
  ];

  buildInputs = [
    glib
    libical
  ];
```